### PR TITLE
Docker and Makefile changes for Stork CBT

### PR DESCRIPTION
**Pull request by Rohit-PX**
Signed-off-by: Rohit-PX <rkulkarni@purestorage.com>

**What type of PR is this?**
integration-test

**What this PR does / why we need it**:
Adding a new jenkins pipeline that will build stork. This will be used for CBT as well as releasing stork.

**Does this PR change a user-facing CRD or CLI?**:
no

**Is a release note needed?**:
no

**Does this change need to be cherry-picked to a release branch?**:
no

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+  stages {
+    stage("Build Stork") {
+      steps {
+        build(job: "${CBT_BUILD_STORK_BUILD_JOB_NAME}", parameters: [string(name: "STORK_GIT_COMMIT", value: GIT_COMMIT)])
+      }
+    }
+    stage("Run Stork CBT tests") {
+      steps {
+        build(job: "${CBT_STORK_JOB_NAME}", parameters: [string(name: "STORK_GIT_COMMIT", value: GIT_COMMIT)])
```
Comment (mborodin-px) [inline Jenkinsfile:16]: rename `CBT_STORK_JOB_NAME` to `CBT_STORK_TEST_JOB_NAME`

**Review by mborodin-px — COMMENTED** [review]
what values should be for these CBT_BUILD_STORK_BUILD_JOB_NAME and CBT_STORK_TEST_JOB_NAME on Jenkins?

**Comment by Rohit-PX** [comment]
> CBT_BUILD_STORK_BUILD_JOB_NAME

CBT_BUILD_STORK_BUILD_JOB_NAME ==> https://jenkins.pwx.dev.purestorage.com/job/CBT/job/cbt-stork-build/
CBT_STORK_TEST_JOB_NAME ==> https://jenkins.pwx.dev.purestorage.com/job/CBT/job/cbt-stork-test/

**Comment by Rohit-PX** [comment]
Successful run after switching to Docker build container: https://jenkins.pwx.dev.purestorage.com/job/CBT/job/cbt-stork-build/75/

**Review by pp511 — APPROVED** [review]
lgtm

**Comment by Rohit-PX** [comment]
Latest successful runs:
Build: https://jenkins.pwx.dev.purestorage.com/job/CBT/job/cbt-stork-build/92/
CBT test: https://jenkins.pwx.dev.purestorage.com/job/CBT/job/cbt-stork-test/112/

**Pull request merged**